### PR TITLE
manually add datepicker w/ npm auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
@Piicksarn @cdnjs/intern2 could you help me review this PR? thanks.
Pull request for issue: #7682

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

# Profile of the lib
 * Git repository (required): https://github.com/fengyuanchen/datepicker
 * Official website (optional, not the repository): https://fengyuanchen.github.io/datepicker/
 * NPM package url (optional): https://www.npmjs.com/package/@fengyuanchen/datepicker
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 170
   - Count of watchers: 13
   - Count of forks: 43
 * NPM download stats (optional):
   - Downloads in the last day: no data
   - Downloads in the last week: no data
   - Downloads in the last month: no data

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
@pvnr0082t if the author only provide minified file, we usually fit the file structures.

**Comment by pvnr0082t** [comment]
@maruilian11 I think we should keep the file structure consistency among different versions. I mean If the author only provide minified files officially in the latest version, we don't need to minify files with *.map in old versions. 
@PeterDaveHello Could you help us clarify this problem?

**Comment by pvnr0082t** [comment]
ping @PeterDaveHello. In a version, if author only provides some minified files, other files should be minified with or without `.map`. Could you help us clarify this problem?

**Comment by PeterDaveHello** [comment]
Both fine.

File: ajax/libs/datepicker/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "npmName": "@fengyuanchen/datepicker",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "i18n",
```
Comment (PeterDaveHello) [inline ajax/libs/datepicker/package.json]: `i18n` should not be a base path.

**Comment by maruilian11** [comment]
@Piicksarn  @cdnjs/intern2 could you help me check this PR? Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
For this situation npm can't get all the versions but git cat, use git auto-update next time.

**Comment by maruilian11** [comment]
got it.

**Pull request merged**